components/brave_shields/content/browser/brave_shields_util.h

```diff
@@ -141,6 +143,12 @@ void SetForgetFirstPartyStorageEnabled(HostContentSettingsMap* map,
 bool GetForgetFirstPartyStorageEnabled(HostContentSettingsMap* map,
                                        const GURL& url);
 
+void SetWebcompatFeatureSetting(HostContentSettingsMap* map,
```

Comment:
add a comment please. Is it only for tests?

Reply:
Comment added.

This is both for tests and for the UI that is in a follow-up PR.